components/compliance-service/habitat/plan.sh

```diff
@@ -38,7 +38,9 @@ pkg_deps=(
   core/jq-static            # Used in habitat/hooks/health_check
   ${local_platform_tools_origin:-chef}/automate-platform-tools
   # WARNING: Update with care. The chef/inspec is managed with Expeditor.
+
   # See .expeditor/update-inspec-version.sh for details
+  # When updating this, also update 'remote_inspec_version' in ./default.toml
```

Comment:
Can we modify the script to update the version pin in the new locations as well so that our existing automation still does the right thing?

Reply:
Yes, please check the most recent commit ( https://github.com/chef/automate/pull/414/commits/e17e1e204e806bf68d0b26ad55ee312818a631f3 ). Is that in line with your thinking?

Reply:
Good catch Steven! Take a look at this ( https://github.com/chef/automate/pull/414/commits/ab45ccb6237861ede6ae040d5c4a669ead00ed46 ) commit I pushed in order for expeditor to continue to match the inspec package.